packages/app/src/explorer/Explorer.module.css

```diff
@@ -64,6 +64,7 @@
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
+  line-height: 1.5; /* fix cropping of ascenders and descenders  */
```

Comment:
![image](https://github.com/silx-kit/h5web/assets/2936402/ae79664a-f828-4785-9db6-559c0de7b61c) 👉  ![image](https://github.com/silx-kit/h5web/assets/2936402/3df6b928-caa2-4235-8f9d-09d2e8d0d9be)

Reply:
The alignment of the text with the arrow changes very slightly, but I think for the better.